### PR TITLE
nis entries in /etc/shadow

### PR DESCRIPTION
**Pull request by stoicbo**
Add support for the nis default entry + and the nis default entry with overrides +:::::::: in /etc/shadow

### CODE REVIEW

File: lenses/shadow.aug [hunk 1]
```diff
@@ -59,11 +59,24 @@ let entry   = [ key word
...
+        colon
+      . [ label "password"          . store word?    . colon ]
+      . [ label "lastchange_date"   . store integer? . colon ]
+      . [ label "minage_days"       . store integer? . colon ]
+      . [ label "maxage_days"       . store integer? . colon ]
+      . [ label "warn_days"         . store integer? . colon ]
+      . [ label "inactive_days"     . store integer? . colon ]
+      . [ label "expire_date"       . store integer? . colon ]
+      . [ label "flag"              . store integer? ] in
+  [ dels "+" . label "@nisdefault" . overrides? . eol ]
```
Comment (raphink) [inline lenses/shadow.aug]: Lines 65-72 can probably be put in common with the `entry` definition here.

**Comment by raphink** [comment]
Thanks for the PR. Can you just factorize the code as stated in my comment please?

**Comment by stoicbo** [comment]
Thanks. Updated per comment.

**Comment by lutter** [comment]
Squashed into one commit, edited commit message slightly and commited as 071175d1

**Pull request closed without merging**